nmxact/xact/fs.go

```diff
@@ -175,6 +176,7 @@ func nextFsUploadReq(s sesn.Sesn, name string, data []byte, off int) (
 	}
 
 	oversize := len(enc) - min(s.MtuOut(), cMaxFSUploadChunk)
+	oversize = 0
```

Comment:
should we just comment out this section instead of setting oversize to 0?

Reply:
I was trying to leave most of the original code intact and given the cascading nature of “variable unused” errors that accumulate as soon as I start commenting out lines, I opted to go this way. If you and @Thenerdstation prefer, I can be convinced to just remove the original lines entirely and have only the simplified logic there (since the original code will still be in the history and the changes of pulling upstream changes in this area is very low).